frontend/playground-frontend/src/pages/feedback.tsx

```diff
@@ -70,20 +68,29 @@ const Feedback = () => {
       email.trim() &&
       feedback.trim()
     ) {
-      /*
-      setSuccessful(
-        await send_feedback_mail(firstName, lastName, email, feedback)
-      );*/
+      if (recaptcha != "") {
+        const emailResult = await sendFeedback({
```

Comment:
@Dashrekker can you attach a brief video to the pr description demoing your feedback form user flow so that others on the team are on the same page

Reply:
thx